pkg/resourceinterpreter/default/native/reflectstatus.go

```diff
@@ -220,6 +222,31 @@ func reflectPodDisruptionBudgetStatus(object *unstructured.Unstructured) (*runti
 	return helper.BuildStatusRawExtension(grabStatus)
 }
 
+func reflectHorizontalPodAutoscalerStatus(object *unstructured.Unstructured) (*runtime.RawExtension, error) {
```

Comment:
Can the default `reflectWholeStatus` support this?

Reply:
yes, it can

Reply:
I think `DesiredReplicas` is also needed.,even if you don't need this field.

Reply:
I think you are right, I will add it~

Reply:
Could you test the scenario of two member clusters?

Reply:
ok, I will update the `pp` to test again~

Reply:
Has this point not been changed yet?

Reply:
> I think DesiredReplicas is also needed.,even if you don't need this field.

done

Reply:
> Could you test the scenario of two member clusters?

done